example_localVerify_test.go

```diff
@@ -101,14 +105,64 @@ func Example_localVerify() {
 	// payload Content: {"targetArtifact":{"mediaType":"application/vnd.docker.distribution.manifest.v2+json","digest":"sha256:c0d488a800e4127c334ad20d61d7bc21b4097540327217dfab52262adc02380c","size":528}}
 }
 
-func createTrustStore() {
+func generateExampleSignatureEnvelope() ([]byte, error) {
+	// exampleSignatureEnvelopePem is a valid signature envelope in COSE format.
+	// It is generated in a previous Sign process and encoded into a pem block.
+	// Users should replace it with their own signature envelope.
+	// Regarding how to generate such signature envelopes, users could refer to
+	// `example_localSign_test.go`.
+	exampleSignatureEnvelopePem := `-----BEGIN SIGNATURE ENVELOPE-----
```

Comment:
It is better to state `EXAMPLE SIGNATURE ENVELOPE`. Otherwise, users may think the signature is a PEM block, which is not.